### DISCUSSION TITLE
26 series simulations

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#198] [TF5: Coupled SE/MOM6]
We started 3 new simulations following the discussion at the [co-chair meeting on Oct 25](https://docs.google.com/presentation/d/1oQV2S0lciEqtRgkoHzcJ0t8WMlhL3Pos/edit?usp=sharing&ouid=110527899490311587217&rtpof=true&sd=true) 

26e = 26c + cice6 
26f = 26c + pertlim = 1.d-14
26g = 26c + pertlim = 2.d-14

In the CAM metrics (as of Monday): 
26e => still too short too tell
26f => a bit on the edge but still ok
26g => looks fine so far 
  
![199097621-4218fe39-6c44-4549-98cc-aa2616d0e52a (1)](https://user-images.githubusercontent.com/9723220/199530493-1b2d3365-72b5-498a-979b-e52b8c2d3783.png)

PS: We will update the plots this morning as we have more years.

@gustavo-marques, @JulioTBacmeister, @adamrher, @dabail10, @fobryan3, @klindsay28

**Comment by cecilehannay** [comment]
Indeed 26f looks bad now:
![199537912-ba35f76d-124e-488d-b381-9e34952cc9bf](https://user-images.githubusercontent.com/9723220/199541096-d05fa9a3-c90f-496d-8d6e-d2e2380b1bb4.png)

**Reply by cecilehannay** [reply]
Thanks !

**Reply by JulioTBacmeister** [reply]
Interesting - so 26e,f,g are just pertlims off of 26c?

**Reply by dabail10** [reply]
Nope. 26e = 26c + CICE6

**Comment by cecilehannay** [comment]
Based on [today's discussion](https://docs.google.com/presentation/d/1gWdG7nGtWXeGIBRg0pHRbwHL53zQdD0F/edit?usp=share_link&ouid=110527899490311587217&rtpof=true&sd=true), we will extend:
26c, 26e, and 26g
